### ISSUE TITLE
Picture Effect not working on some pictures (Windows 2022.23)

### ISSUE THREAD

**Issue by tpthatsme** [smeighan/xLights#3595]
**Describe the bug**
Picture effect on a Mega Tree does not display as expected regardless of scale options or positions. Interestingly, some GIFs work properly and some do not. Also the GIF works as expected if the tree is split into horizontal sub-models.
Image used during testing: https://media0.giphy.com/avatars/Kool-Aid/vBEBR9AYEYqq.gif

**To Reproduce**
1. Create new sequence
2. Create a tree
3. Put a Picture effect on the tree with the link from the description
4. Select Shrink to fit

You should notice that the image plays as expected in Windows 2022.21, but does not in Windows 2022.23

**Expected behavior**
Animated GIF displayed on the Mega Tree

**Screenshots**
See attached for expected results (version 2022.21). Version 2022.23 does not display the GIF.

**Versions (please complete the following information):**
 - OS: Windows 11
 - xLights version 2022.23

**Additional context**

**Attachments**
[2022.23 bug.zip](https://github.com/smeighan/xLights/files/10097105/2022.23.bug.zip)

**Comment by derwin12** [comment]
Seems to work in .24

**Comment by tpthatsme** [comment]
@derwin12 
> Seems to work in .24

I am sorry, but I disagree. Please see the following video where it is working in version 22 but not working in version 24.
https://youtu.be/8dRyFCpzOak

**Comment by derwin12** [comment]
Weird .. works here .. wonder what the difference is..

https://user-images.githubusercontent.com/4643499/205472014-dc01335b-2177-4807-bc9c-54b3764e22f2.mp4

**Comment by keithsw1111** [comment]
The problem is scaling. See the vertical bars in this image ... certain scaling values result in a black image.

![image](https://user-images.githubusercontent.com/10283944/205494602-3a1d2ea4-b288-4a0f-bf2f-f85a0a3895f8.png)

**Comment by derwin12** [comment]
Just realized that my ACDC sequence is not showing Angus dancing on the mega tree.. Going back to .10 to get him back.
(Moving resized gif image)

**Comment by derwin12** [comment]
Realized my show sequences were missing pictures .. no easy way to find all the missing ones.
Rolling back to 22 was the fix. The attached is from ACDC sequence. Nothing shows in .23 or .24
Just an empty mega tree :-(

[GifBug.zip](https://github.com/smeighan/xLights/files/10200241/GifBug.zip)

**Comment by AzGilrock** [comment]
I wanted to investigate this but I can't get it to fail.  It's working on Mac and Windows for me.  Here's Daryl's sequence:
<img width="664" alt="Screenshot 2022-12-11 at 10 27 23 AM" src="https://user-images.githubusercontent.com/2024669/206918968-f2bfe504-adbb-4eb7-bc9b-2dff76b3a0b7.png">

**Comment by dkulp** [comment]
It's only a Windows issue (does not exist on Linux or Mac) and only with release mode builds.   It's an optimizer problem with Visual Studio.  I don't know if there is an easy way to rebuild wxWidgets with a lower optimization setting (equivalent of -O1 instead of -O2) or if that would help or not.

**Comment by AzGilrock** [comment]
Yeah but it worked in the release version of .24 for me on Windows also.  Maybe the matrix I created didn't cause the right scaling that makes it fail.

**Comment by AzGilrock** [comment]
I created the same size tree Daryl has on Windows release 2022.24 and like I original said I can't get it to fail.

![image](https://user-images.githubusercontent.com/2024669/206921369-814726f0-85e0-4375-95ed-4669445ad480.png)

**Comment by computergeek1507** [comment]
What version of vs are you using, Keith switched to 17.4 for 23 and 24. I have 17.4 on my laptop and it doesn't work for me. The GitHub action still uses 17.3 so I wonder if the GitHub build works or not

**Comment by computergeek1507** [comment]
I built wx on 17.4 with o1 optimization and it didn't work

**Comment by AzGilrock** [comment]
Well I haven't been building it on Windows lately I just run release versions on Windows for testing.  I have VS 2022, VS 2013, and VS 2008 all installed.

**Comment by derwin12** [comment]
Anything I can do to identify the images that wont work? 2023.01 - no difference. I cant figure out how I will find all the possible broken images.

**Comment by AzGilrock** [comment]
Well I ever could make anything fail.  I gotta have it fail on my setup or I can’t help.

**Comment by drakkar20d** [comment]
seems No scaling and Scale to keep aspect ratio  don't work , Scale to fit and Scale to keep aspect ratio crop do work but the images don't look as they should in 2023.1

**Comment by derwin12** [comment]
FYI the attached GifBug.zip is *working* in the latest xlights build .. 2023.06.
